ui/src/components/flatpicker/FlatPicker.js

```diff
@@ -90,6 +90,9 @@ export default class FlatPicker extends React.Component {
         flatpickr('.' + fpClass, {
             onChange: this.onChange,
             onClose: this.onClose,
+            onReady: (dates, dateStr, instance) => {
+                instance.altInput.id = 'member-expiry';
```

Comment:
this would give every flatpickr the id "member-expiry" which is not the case since we also use flatpickr for history dates

Reply:
based on line 111, it does have data-testid='flatPicker', we cannot use that?